vizro-ai/docs/pages/user_guides/run_vizro_ai.md

```diff
@@ -80,3 +80,18 @@ Vizro-AI's `_get_chart_code` method returns the Python code string that can be u
 
 The returned `code_string` can be used to dynamically render charts within your application. You may have the option to encapsulate the chart within a `fig` object or convert the figure into a JSON string for further integration.
 In case you would like to use the insights or code explanation, you can use `vizro_ai._run_plot_tasks(df, ..., explain=True)`, which returns a dictionary containing the code explanation and chart insights alongside the code.
+
+### 4. `max_debug_retry` Parameter in plot Function
+Since the code is generated by Large Language Models (LLMs), it can contain bugs and thus fail on execution.
+When an error is encountered, an LLM-based debugging process will be triggered.
+The `max_debug_retry` parameter controls maximum number of retries the `plot` function attempts to debug errors.
+
+- **Default Value**: 3
+- **Type**: int
+- **Brief**: By default, the `max_debug_retry` is set to 3, the function will attempt to debug errors up to three times.
+If the errors are not resolved after the maximum number of retries, the function will cease further debugging attempts.
+- **Example**:
+   Adjust the number of retries by setting `max_debug_retry` in the `plot` function.
+    ```py
+     vizro_ai.plot(df = df, user_input = "your user input", max_debug_retry= 5)
+    ```
```

Comment:
Due to github formatting I cannot propose smth here, but I think we can make this section even snappier.

If we move this to an API, we ideally want this to be automatically populated, so I think there is no need to describe it in detail here.

We also avoid to much bolding in general, but this is a somewhat unofficial guideline.

Would something like:
----
E.g. if you would like 5 debugging attempts, you can set `max_debug_retry=5` in the `plot` function:

```py
 vizro_ai.plot(df = df, user_input = "your user input", max_debug_retry= 5)
```
----
be good?